src/common/SPELLS/rogue.js

```diff
@@ -193,12 +198,12 @@ export default {
   },
   SHADOW_SATYRS_WALK_ENERGY_BASE: {
     id: 224914,
-    name: 'Shadow Satyr\'s Walk',
+    name: "Shadow Satyr's Walk",
```

Comment:
the standard across everything is the way it was
'Shadow Satyr\'s Walk'

Just so everything is uniform. probably not a big deal, but that is how everything is formatted